lisa/sut_orchestrator/azure/features.py

```diff
@@ -2300,24 +2300,10 @@ def create_setting(
             ):
                 capabilities.append(SecurityProfileType.SecureBoot)
         # https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview # noqa: E501
-        if cvm_value and resource_sku.family.casefold() in [
-            "standarddcasv5family",
-            "standarddcadsv5family",
-            "standardecasv5family",
-            "standardecadsv5family",
-            "standarddcev5family",
-            "standarddcedv5family",
-            "standardecev5family",
-            "standardecedv5family",
-        ]:
+        if cvm_value.casefold() == "snp":
```

Comment:
Update comments above.

Reply:
Do you just want me to remove the link to CVM overview?

Reply:
The original link was the source of vm size families. It looks the content was moved, so the link should be removed too.